packages/plugins/ansible-collection-theforeman-foreman/ansible-collection-theforeman-foreman.spec

```
@@ -53,8 +58,25 @@ cp -a ./* %{buildroot}%{collection_directory}
 %doc %{collection_directory}/README.md
 
 
+%package documentation
+Summary:    Documentation for Foreman Project Ansible modules collection
+Source1:    %{collection_namespace}-%{collection_name}-%{version}.tar.gz
+
+
+%description documentation
+Documentation for collection of Ansible Modules to manage Foreman installations.
+Includes modules for Katello.
+
+
+%files documentation
+%doc %dir /usr/share/doc/%{name}
+/usr/share/doc/%{name}
+
 
 %changelog
+* Mon Jul 25 2022 Maximilian Kolb - 3.5.0-1
```

You should increase the release and then use this version. Then at least CI can build it and we can see the error
```suggestion
* Mon Jul 25 2022 Maximilian Kolb - 3.4.0-2
```